epoll.js

```diff
@@ -1,2 +1,9 @@
-module.exports = require('bindings')('epoll.node');
-
+module.exports = (() => {
+  const osType = require('os').type();
+  if (osType == 'Linux') {
+    require('bindings')('epoll.node');
+  } else {
```

Comment:
If this is working on Linux, why don't we need to return the require statement here?

Reply:
I doesn't work on Linux.